src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -327,6 +327,39 @@ export default defineConfig({
 });
 ```
 
+## Experimental features
+
+The following features are also available for use, but may be subject to breaking changes in future updates. Please follow the [`@astrojs/netlify` CHANGELOG](https://github.com/withastro/astro/tree/main/packages/integrations/netlify/CHANGELOG.md) carefully for updates if you are using these features in your project.
+
+### `experimentalStaticHeaders`
+
+<p>
+  **Type:** `boolean` <br />
+  **Default:** `false`<br />
+  <Since v="6.5.0"  pkg="@astrojs/netlify"/>
+</p>
+
+Enables specifying custom headers for prerendered pages in Netlify's configuration.
+
+If enabled, the adapter will save [static headers in the Framework API config file](https://docs.netlify.com/frameworks-api/#headers) when provided by Astro features, such as Content Security Policy.
+
+For example, when [experimental Content Security Policy](/en/reference/experimental-flags/csp/) is enabled, `experimentalStaticHeaders` can be used to add the CSP `headers` to your Netlify's configuration, instead of creating a `<meta>` element:
```

Comment:
```suggestion
For example, when [experimental Content Security Policy](/en/reference/experimental-flags/csp/) is enabled, `experimentalStaticHeaders` can be used to add the CSP `headers` to your Netlify configuration, instead of creating a `<meta>` element:
```